### PR TITLE
Add documentation to enable or disable a client

### LINKED ISSUE #115: Add documentation on how an admin can disable a client
Related to PR https://github.com/indigo-iam/iam/pull/747.

### PR DESCRIPTION
**Pull request by garaimanoj**

### CODE REVIEW

File: content/en/docs/tasks/administration/client-status-change/_index.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+To confirm your choice click on _Disable Client_ button on the modal window:
+
+![disable client modal](../images/client-status-change-4.png)
+
+On success you will get a confirmation message and the _Disable Client_ button will change to _Restore Client_:
+
+![disable client confirmation](../images/client-status-change-5.png)
+
+Using _Restore Client_ button client's status can be change back to _Active_.
```
Comment (rmiccoli) [inline content/en/docs/tasks/administration/client-status-change/_index.md]: Fix typos: **client status** can be **changed**

File: content/en/docs/tasks/administration/client-status-change/_index.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
+---
+title: Disable or Enable a client
+weight: 2
+---
+Existing clients can be disabled or enabled by admin user from dashboard.
+
+## Disabling a client using the dashboard
+
+Log into the service using admin credential and click on the _Clients_ link on the left
```
Comment (rmiccoli) [inline content/en/docs/tasks/administration/client-status-change/_index.md]: admin credentials*

**Comment by rmiccoli** [comment]
Looks good to me :)

File: content/en/docs/tasks/administration/client-status-change/_index.md (hunk 1 above)
Comment (garaimanoj) [reply content/en/docs/tasks/administration/client-status-change/_index.md]: Thanks. Fixed.

File: content/en/docs/tasks/administration/client-status-change/_index.md (hunk 2 above)
Comment (garaimanoj) [reply content/en/docs/tasks/administration/client-status-change/_index.md]: Thanks. Fixed.

**Review by rmiccoli — APPROVED** [review]

**Review by federicaagostini — APPROVED** [review]

**Pull request merged**